src/macros.js

```diff
@@ -895,6 +896,17 @@ defineMacro("\\argmin", "\\DOTSB\\operatorname*{arg\\,min}");
 defineMacro("\\argmax", "\\DOTSB\\operatorname*{arg\\,max}");
 defineMacro("\\plim", "\\DOTSB\\mathop{\\operatorname{plim}}\\limits");
 
+//////////////////////////////////////////////////////////////////////
+// derivative.sty
+// https://ctan.math.illinois.edu/macros/latex/contrib/derivative/derivative.pdf
+
+defineMacro(
+    "\\dv", (context) => assembleDerivativeExpr("\\textnormal{d}", context));
+defineMacro(
+    "\\odv", (context) => assembleDerivativeExpr("\\textnormal{d}", context));
+defineMacro(
+    "\\pdv", (context) => assembleDerivativeExpr("\\partial", context));
+
```

Comment:
I think it would make more sense to move these side-effects into `src/macros/derivatives.js`, which would then `import defineMacro from "./defineMacro"`.

However, we haven't decided to make a `src/macros` directory. Perhaps you could provide your input on #2994? What do you think of putting your code in `src/functions/derivatives.js`, and possibly renaming `functions` to `commands`?